app/src/test/java/com/brewfind/BreweryDataUnitTest.kt

```diff
@@ -37,7 +36,7 @@ class BreweryDataUnitTest {
     }
 
     private fun whenSearchForStreetside() {
-        mvm.fetchBreweries("Streetside")
+        mvm.fetchBreweriesByName("Streetside")
```

Comment:
Fixing an error which would call a nonexistent string in another method. Changed it by having it call a new method with the purpose of having the string that was needed.